src/main/java/net/fabricmc/loom/api/mappings/layered/MappingsNamespace.java

```diff
@@ -37,6 +37,16 @@ public enum MappingsNamespace {
 	 */
 	OFFICIAL,
 
+	/**
+	 * Official names for the Minecraft client jar, usually obfuscated.
```

Comment:
Nit: Maybe suggest this is only used pre 1.3, when the client/server had differing names.